src/analyzer/plugins/analyzerqueenmarykey.cpp

```diff
@@ -61,8 +61,9 @@ bool AnalyzerQueenMaryKey::initialize(mixxx::audio::SampleRate sampleRate) {
             windowSize, stepSize, [this](double* pWindow, size_t) {
                 int iKey = m_pKeyMode->process(pWindow);
 
-                VERIFY_OR_DEBUG_ASSERT(ChromaticKey_IsValid(iKey)) {
+                if (!ChromaticKey_IsValid(iKey)) {
                     qWarning() << "No valid key detected in analyzed window:" << iKey;
+                    DEBUG_ASSERT(false);
```

Comment:
I know that you log a message immediate before this assertion, but maybe the order becomes confusing due to logging from multiple threads or something.

I suggest that you put a message into the assertion like this:


```suggestion
                    DEBUG_ASSERT(!"chromatic key is invalid");
```